### PR TITLE
Feature/service yml configs

### PR DESCRIPTION
**Pull request by outrunthewolf**
Added functionality:

 - Launch Vessel
 - Launch Fleet
 - Build Vessel
 - Build Fleet
 - Scuttle Vessel
 - Scuttle Fleet

### CODE REVIEW

File: src/Commands/Command.php [hunk 1]
```diff
@@ -17,4 +24,305 @@ public function __construct(HarbourYML $yml, $command)
...
+            $productionComposeFilePath = $localRepoLocation . '/' . $config['service_compose_production_location'] . '/' . $config['service_compose_filename'];
+            if(!file_exists($productionComposeFilePath)) {
+                $this->writeln($output, 'No valid ' . $config['service_compose_filename'] . ' file found for: ' . $name, 'error');
+                $this->writeln($output, 'Looking for file in: ' . $productionComposeFilePath, 'error');
+                $this->writeln($output, '');
+                $prodCompose = false;
+            }
+
+            // If both compose files are found we can continue
+            if($devCompose == true && $prodCompose == true) {
```
Comment (zot24) [inline src/Commands/Command.php]: What happen if we want to run Harbour in `prod`? do we need the `devp` compose as well?

File: src/Commands/Command.php [hunk 2]
```diff
@@ -17,4 +24,305 @@ public function __construct(HarbourYML $yml, $command)
...
+     *
+     * @param string $service The name of the service
+     * @param string $file The path of the copose file
+     */
+    protected function launchService($service, $file)
+    {
+        $command = "docker-comp -p $service -f $file up -d";
+
+        $process = new Process($command);
+        $process->setTimeout(3600);
```
Comment (zot24) [inline src/Commands/Command.php]: 👍

**Comment by zot24** [comment]
Not sure about differentiate between Build and Launch it's kind of Launch depend on Build but it's most likely we'll never Build without a Launch plus this scenario will only happen in development on production we'll never Build just Launch (because our image will be already builded)

**Comment by zot24** [comment]
Other thing it's the `Command` class is a mess a will move all those commands to a Helper for each `category` like `Helpers/FleetHelper.php`/`Helpers/VesselHelper.php` or a Trait for each instead of Helpers just to keep concerns separated

Apart of those two thin Looks good! does it work? 😊

**Comment by outrunthewolf** [comment]
of course it works you animal

File: src/Commands/Command.php (hunk 1 above)
Comment (outrunthewolf) [inline src/Commands/Command.php]: I think production will be another consideration I just want local working right now.

**Pull request merged**